docs/tutorials/image-modality.md

```diff
@@ -0,0 +1,54 @@
+# Training/Inference with Image Modality
+If your training and inference tasks involve image modality, your data would contain a column representing the path to the image file, i.e.
+
+```python
+   feature_1                     image   label
+0          1   image/train/train_1.png       0
+1          2   image/train/train_1.png       1
+```
+
+### Preparing the Image Column
+Currently, AutoGluon only supports one image per row.
```

Comment:
I think we support multiple images now https://github.com/autogluon/autogluon/blob/master/docs/tutorials/multimodal/multimodal-faq.md#how-does-autogluon-multimodal-handle-multiple-images-per-sample